go/runtime/config/config.go

```diff
@@ -269,6 +272,24 @@ func (c *ComponentConfig) UnmarshalYAML(value *yaml.Node) error {
 	}
 }
 
+// NetworkingConfig is the networking configuration.
+type NetworkingConfig struct {
+	// Incoming is a list of IPs/ports to expose to the component from the host.
+	Incoming []IncomingNetworkingConfig `yaml:"incoming,omitempty"`
+}
+
+// IncomingNetworkingConfig describes an IP/port to expose to the component from the host.
+type IncomingNetworkingConfig struct {
+	// IP is the host IP address to expose to the component.
+	IP string `yaml:"ip,omitempty"`
+	// Protocol is the optional protocol to expose to the component.
+	Protocol string `yaml:"protocol,omitempty"`
+	// SrcPort is the source port (on the host).
+	SrcPort uint16 `yaml:"src_port"`
```

Comment:
If you want to forward multiple ports to the same IP, then you make multiple `IncomingNetworkConfig` sections right?

Reply:
Yeah.